Source/com/drew/metadata/exif/makernotes/FujifilmMakernoteDescriptor.java

```diff
@@ -466,4 +471,11 @@ public String getDynamicRangeSettingDescription()
                 return "Unknown (" + value + ")";
         }
     }
+    @Nullable
+    public int getImageNumber()
+    {
+        final Integer value = _directory.getInteger(TAG_IMAGE_NUMBER);
+        return value;
+    }
```

Comment:
Methods on the `*Descriptor` classes generally return a `String` description. This method feels inconsistent with the rest of the API here. What do you think?

Reply:
Is there a benefit of returning String descriptions exclusively? If it's a matter of feel, aesthetics, style, then I'd get that and it's up to you, really.

Reply:
The `Descriptor` classes are supposed to return presentable string versions. A method that gets a tag's raw value would be a better fit on the `Directory` class, following the design that exists elsewhere here.

Apart from moving to the `Directory`, this method will throw a `NullPointerException` if the tag's not present in the image, from the implicit conversion from `Integer` to `int`. The `@Nullable` attribute doesn't apply to `int` either.

I think I would vote to remove this method. We generally don't have methods for fields like this anywhere else in the API apart from where they have some complex logic such as converting from an encoded representation into some other form that's useful (and not just a string description). In this case it's no different than any other integer tag.

Reply:
You're correct on all counts.